tests/console/test_select_option.py

```diff
@@ -22,6 +22,26 @@ def test_select_option():
     assert result == 'second'
 
 
+def test_select_option_list():
```

Comment:
Per previous comment, I'd prefer we change `option_list` to always be a list of tuples.

Reply:
ack, but respectfully disagree in this case.